.github/release-please/manifest.json

```diff
@@ -1,5 +1,3 @@
 {
-  "dsp-domain": "2.2.0",
-  "dsp-meta": "2.2.0",
-  "dsp-meta-cmd": "2.2.0"
+  ".": "2.2.0"
```

Comment:
Do we need manifest file at all now if the workspace was removed? According to [this](https://github.com/googleapis/release-please?tab=readme-ov-file#strategy-language-types-supported), manifest is necessary for manifest driven releases (with workspace). Version is kept in cargo file.

Reply:
The documentation on how to setup a rust project is quite slim, all I have found was the link you have provided, where it says "it is not required". However, when [I tried setting it up with the command line tool](https://github.com/dasch-swiss/dsp-meta/pull/201) it generated the manifest and config for me.

I am not entirely sure if this setup is working for us unless I try it out I guess.